go.mod

```diff
@@ -23,8 +23,11 @@ require (
 	github.com/jbowtie/ratago v0.0.0-20200401224626-3140c0a9b186
 	github.com/kofalt/go-memoize v0.0.0-20220914132407-0b5d6a304579
 	github.com/magefile/mage v1.14.0
+	github.com/mattn/go-isatty v0.0.17
 	github.com/milosgajdos/tenus v0.0.3
+	github.com/pkg/errors v0.9.1
 	github.com/quic-go/quic-go v0.40.1
+	github.com/shirou/gopsutil v3.21.11+incompatible
```

Comment:
I don't think we need a big dependency with some transitive dependencies for simply checking 1 file